packages/manager/apps/pci-savings-plan/src/components/Chart/Chart.tsx

```diff
@@ -142,15 +141,15 @@ const GenericChart: React.FC<ChartProps> = ({
             type="step"
             dataKey="excluded"
             stackId="2"
-            stroke="#FFC0CB"
-            fill={EXCLUDED_COLOR}
+            fill="#FFCCD9"
```

Comment:
As you made for `stroke` variables, you could encapsulate this one into a constant to gain consistency :)